### PR TITLE
Upgrade to react 15.5.0

### PR DESCRIPTION
**Pull request by madflanderz**
Fixes this issue: #8 

I needed a clean project and so i had to fixed all errors and warnings coming from current react version. 
Everything works fine except the npm run test, but i have no idea what gets wrong. Maybe you have a clue whats the problem there.

### CODE REVIEW

File: src/react-toggle-button.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types'
```
Comment (AndreiRailean) [inline src/react-toggle-button.js:2]: 👍

**Review by AndreiRailean — APPROVED** [review]
Looks good to me. Should be merged soon to avoid falling behind. This repo doesn't have many tests, so knowing that anything beyond just rendering it out is breaking is hard without just loading it in the browser and interacting with it.

**Comment by wesrobinson12** [comment]
Any update on this? Would like to get this working with React 15.5+

**Comment by gdowens** [comment]
closing this, just going to make the upgrade to react16 right now.

**Pull request closed without merging**